akka-stream-tests/src/test/scala/akka/stream/DslFactoriesConsistencySpec.scala

```diff
@@ -46,6 +46,7 @@ class DslFactoriesConsistencySpec extends WordSpec with Matchers {
       (classOf[scala.Function0[_]],                        classOf[java.util.concurrent.Callable[_]]) ::
       (classOf[scala.Function1[_, Unit]],                  classOf[akka.japi.function.Procedure[_]]) ::
       (classOf[scala.Function1[_, _]],                     classOf[akka.japi.function.Function[_, _]]) ::
+      (classOf[scala.Function1[_, _]],                     classOf[java.util.function.BiFunction[_, _, _]]) :: // setup
```

Comment:
Function1  vs BiFunction? should it be Function2?

Reply:
It was because on the Scala API it was Function1 to Function1 to ... and in Java it was BiFunction. Will update the Scala API to be the same as in Java.